### ISSUE TITLE
Does the Chamelon dataset only contain testing set?

### ISSUE THREAD

**Issue by Johann356** [shilinyan99/AIDE#13]
I didn't see any mention of Chamelon training set in the paper. So does the dataset contain training and testing set split?

**Comment by shilinyan99** [comment]
Thank you for your interest in our work. Please note that Chameleon is solely a test set and does not contain a corresponding training set.
